contentgrid-spring-data-rest/src/main/java/com/contentgrid/spring/data/rest/webmvc/blueprint/AttributeRepresentationModel.java

```diff
@@ -0,0 +1,78 @@
+package com.contentgrid.spring.data.rest.webmvc.blueprint;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import java.util.Collection;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.RepresentationModel;
+import org.springframework.hateoas.server.core.EmbeddedWrapper;
+import org.springframework.hateoas.server.core.EmbeddedWrappers;
+import org.springframework.hateoas.server.core.Relation;
+
+@Builder
+@Getter
+@AllArgsConstructor
+@Relation(BlueprintLinkRelations.ATTRIBUTE_STRING)
+public class AttributeRepresentationModel extends RepresentationModel<AttributeRepresentationModel> {
+
+    @NonNull
+    private final String name;
+    @JsonInclude(Include.NON_EMPTY)
+    private final String title;
+
+    @NonNull
+    private final String type;
+
+    private final String description;
+
+    @JsonInclude(Include.NON_DEFAULT)
```

Comment:
Ranec pointed out that some keys are only present when it's value is non-null or true. You might easily miss that these keys exist when you're creating a client. (and so you miss information that could be essential)